api/v1beta2/network_types.go

```diff
@@ -29,6 +29,9 @@ type NetworkStatus struct {
 
 	// APIServerELB is the Kubernetes api server classic load balancer.
 	APIServerELB ClassicELB `json:"apiServerElb,omitempty"`
```

Comment:
I'm wondering if it would make sense to pull these together and have a single LoadBalancerSpec and then still do the switching based on the Type. I think I'll do that.